{"message":"Test error","stack":"Error: Test error\n    at D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\vividly\\routes\\genre.js:17:15\n    at Layer.handle [as handle_request] (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at Layer.handle [as handle_request] (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:281:22\n    at Function.process_params (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at Layer.handle [as handle_request] (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at trim_prefix (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:317:13)\n    at D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:284:7\n    at Function.process_params (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:335:12)\n    at next (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\node_modules\\body-parser\\lib\\read.js:130:5","level":"error","timestamp":"2023-08-30T20:44:32.046Z"}
{"message":"Outside Express error","stack":"Error: Outside Express error\n    at Object.<anonymous> (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\vividly\\index.js:28:7)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47","level":"error","timestamp":"2023-08-30T22:08:10.054Z"}
{"message":"Outside Express error","stack":"Error: Outside Express error\n    at Object.<anonymous> (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\vividly\\index.js:28:7)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47","level":"error","timestamp":"2023-08-30T22:08:30.137Z"}
{"message":"rejected error","stack":"Error: rejected error\n    at Object.<anonymous> (D:\\Project\\WebPractice\\Nodejs\\authentication-authorization\\vividly\\index.js:33:26)\n    at Module._compile (node:internal/modules/cjs/loader:1254:14)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)\n    at Module.load (node:internal/modules/cjs/loader:1117:32)\n    at Module._load (node:internal/modules/cjs/loader:958:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:23:47","level":"error","timestamp":"2023-08-30T22:34:24.467Z"}
